# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

bool curvature_available
bool velocity_available
bool is_forward_trajectory

geometry_msgs/Pose2D pose
ExtendedPose2d[] trajectory
